src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String REGEX = "\\D+";
+    private int supplyCount = 0;
+    private int buyCount = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        getCounts(fromFileName);
+        int resultCount = supplyCount - buyCount;
+        writeToFile(toFileName,
+                    "supply," + this.supplyCount + "\n"
+                            + "buy," + this.buyCount + "\n"
+                            + "result," + resultCount + "\n");
```

Comment:
use StringBuilder instead of concatenation

and please use System.lineSeparator()